src/components/playerCharacter/index.ts

```diff
@@ -417,10 +421,18 @@ export const playerCharacter = (
   // OXYGEN
   const consumeOxygen = (delta: number, pressure: number): void => {
     // TODO: as pressure increases, increase rate of consumption
-    state.oxygen -= PLAYER_OXYGEN_CONSUMPTION_RATE;
+    state.oxygen -= delta * PLAYER_OXYGEN_CONSUMPTION_RATE;
 
     if (state.oxygen < 0) {
       state.oxygen = 0;
+
```

Comment:
moved these from the lose screen because there was a delay. these fire immediately now